x/examples/fetch-speed/main.go

```diff
@@ -77,18 +67,12 @@ func main() {
 		if err != nil {
 			return nil, fmt.Errorf("invalid address: %w", err)
 		}
-		if overrideHost != "" {
-			host = overrideHost
-		}
-		if overridePort != "" {
-			port = overridePort
-		}
 		if !strings.HasPrefix(network, "tcp") {
 			return nil, fmt.Errorf("protocol not supported: %v", network)
 		}
 		return dialer.DialStream(ctx, net.JoinHostPort(host, port))
 	}
-	httpClient := &http.Client{Transport: &http.Transport{DialContext: dialContext}, Timeout: 5 * time.Second}
+	httpClient := &http.Client{Transport: &http.Transport{DialContext: dialContext}, Timeout: *timeoutFlag * time.Second}
```

Comment:
The timeout flag is already a Duration:
```suggestion
	httpClient := &http.Client{Transport: &http.Transport{DialContext: dialContext}, Timeout: *timeoutFlag}
```